lib/popcorn/README.md

```diff
@@ -0,0 +1,794 @@
+# Popcorn
+
+**Why endure plain corn when you can pop it?!**
+
+Popcorn is a minimal yet powerful nit web application framework that provides cool
+features for lazy developpers.
```

Comment:
Nha, I'm definitely more lazy than cool. Does it make me cool? Too lazy to think about it...